### PR TITLE
Create home screen #7

### PR DESCRIPTION
**Pull request by AlaaYassin1**

### CODE REVIEW

File: lib/components/BottomNavigationBar/bottomNavigation_controller.dart [hunk 1]
```diff
@@ -0,0 +1,16 @@
+import '../../general_exports.dart';
+
+class BottomNavigationWidget extends GetxController {
+  int currentindex = 0;
```
Comment (alaaalzibda) [inline lib/components/BottomNavigationBar/bottomNavigation_controller.dart:4]: Rename it to `currentindex`

File: lib/components/BottomNavigationBar/bottom_navigation.dart [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+              selectedItemColor: Colors.deepPurpleAccent,
+              currentIndex: controller.currentindex,
+              onTap: controller.onTap,
+              items: const <BottomNavigationBarItem>[
+                BottomNavigationBarItem(
+                  icon: Padding(
+                    padding: EdgeInsets.only(bottom: 6),
+                    child: Icon(Icons.grid_view_rounded),
+                  ),
+                  label: 'Home',
```
Comment (alaaalzibda) [inline lib/components/BottomNavigationBar/bottom_navigation.dart]: Why these words not translated

File: lib/components/BottomNavigationBar/bottom_navigation.dart [hunk 3]
```diff
@@ -0,0 +1,63 @@
...
+              type: BottomNavigationBarType.fixed,
+              showUnselectedLabels: true,
+              unselectedItemColor: Colors.grey,
+              selectedItemColor: Colors.deepPurpleAccent,
+              currentIndex: controller.currentindex,
+              onTap: controller.onTap,
+              items: const <BottomNavigationBarItem>[
+                BottomNavigationBarItem(
+                  icon: Padding(
+                    padding: EdgeInsets.only(bottom: 6),
```
Comment (alaaalzibda) [inline lib/components/BottomNavigationBar/bottom_navigation.dart]: Why using numbers ? not percentage ?

**Review by alaaalzibda — CHANGES_REQUESTED** [review]

File: lib/components/BottomNavigationBar/bottomNavigation_controller.dart (hunk 1 above)
Comment (AlaaYassin1) [reply lib/components/BottomNavigationBar/bottomNavigation_controller.dart:4]: its the same

**Pull request merged**